PoseEstimation-TFLiteSwift/PoseEstimator.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  PoseEstimator.swift
+//  PoseEstimation-TFLiteSwift
+//
+//  Created by Doyoung Gwak on 2020/03/14.
+//  Copyright © 2020 Doyoung Gwak. All rights reserved.
+//
+
+import CoreVideo
+
+enum PoseEstimationResult {
+    case success(keypoints: [Keypoint], heatmaps: Heatmaps)
+    case fail
+    
+    struct Keypoint {
+        let position: CGPoint
+        let score: CGFloat
+    }
+    struct Heatmaps {
+        // <#TODO#>
+        
+        init(tfliteResult: TFLiteResult) {
+            // <#TODO#>
+        }
+        
+        var keypointsAndScore: [(position: CGPoint, score: CGFloat)] {
```

Comment:
Keypoint have a score.
So, keypointsAndScore have duplicated meaning.

And I can implement, var keypoints: [Keypoint] { ... }.
This is more simple, isn't it?

Reply:
@syjdev 
👍 Right, I replaced to `keypoints: [Keypoints]` property.